src/test/scala/nl.knaw.dans.easy.deposit/SubmitterSpec.scala

```diff
@@ -103,12 +104,38 @@ class SubmitterSpec extends TestSupportFixture with MockFactory {
     val depositDir = createDeposit(datasetMetadata.copy(messageForDataManager = None))
     addDoiToDepositProperties(getBag(depositDir))
 
-    createSubmitter(userGroup).submit(depositDir) shouldBe a[Success[_]]
+    val bagStoreBagId1 = succeedingSubmit(depositDir)
 
-    (testDir / "submitted" / depositDir.id.toString / "bag" / "metadata" / "message-from-depositor.txt")
+    (testDir / "submitted" / bagStoreBagId1 / "bag" / "metadata" / "message-from-depositor.txt")
       .contentAsString shouldBe ""
   }
 
+  it should "overwrite a previous bag-store.bag-id at resubmit" in {
+
+    val depositDir = createDeposit(datasetMetadata.copy(messageForDataManager = None))
+    addDoiToDepositProperties(getBag(depositDir))
+
+    // submit twice
+    val bagStoreBagId1 = succeedingSubmit(depositDir)
+    val depositProps = depositDir.bagDir.parent / "deposit.properties"
+    depositProps.write(depositProps.contentAsString.replace("SUBMITTED", "DRAFT"))
```

Comment:
shouldn't `"DRAFT"` be `"REJECTED"`? Don't remember what the picture by @lindareijnhoudt said.

Reply:
i agree. 
![image](https://user-images.githubusercontent.com/2812779/56116921-45460e00-5f67-11e9-98ad-322e4007e8d4.png)
REJECTED -> DRAFT is when the user opens it and saves it